lib/OxShared.ml

```diff
@@ -7,8 +7,6 @@ open OpenFlow0x01
 module Log = Async_OpenFlow.Log
 
 let _ = Log.set_level `Info
-let _ = Log.set_output
-          [Log.make_filtered_output [("ox", "toplevel")]]
```

Comment:
Yeah, it looks like we can't filter anymore. But, who cares, really?

If we want nicer logging, we should consider one of the other packages on OPAM.